winit-core/src/window.rs

```diff
@@ -850,12 +870,15 @@ pub trait Window: AsAny + Send + Sync + fmt::Debug {
     /// - **iOS / Android / Web / Orbital:** Unsupported.
     fn set_surface_resize_increments(&self, increments: Option<Size>);
 
-    /// Modifies the title of the window.
+    /// The window transparency state.
+    ///
+    /// Getter for [`set_transparent`][Window::set_transparent], see that for details.
     ///
     /// ## Platform-specific
     ///
-    /// - **iOS / Android:** Unsupported.
-    fn set_title(&self, title: &str);
+    /// - **iOS / Android / Web:** Unsupported.
+    /// - **X11:** Unimplemented, returns `false`.
```

Comment:
```suggestion
    /// - **X11:** Unimplemented. Returns `false`.
```
I propose we use two sentences. Was told before by people who know better that in English you don't use commas a whole lot, but what do I know.